test/Riok.Mapperly.Tests/Mapping/ObjectPropertyUseNamedMappingTest.cs

```diff
@@ -683,4 +683,73 @@ public record B(int Value);
             .HaveDiagnostic(DiagnosticDescriptors.ReferencedMappingNotFound, "The referenced mapping named MapValue was not found")
             .HaveAssertedAllDiagnostics();
     }
+
+    [Fact]
+    public Task ShouldSupportExternalMappingsOnStaticMapping()
```

Comment:
Also add tests referencing methods on external mappers that don't exist or are not accessible.

Reply:
Added tests for non-existing references.